### DISCUSSION TITLE
Problems with midi2lr on new Macbook Pro M1 Pro with Monterey

### DISCUSSION THREAD

**Discussion by jimvetter** [rsjaffe/MIDI2LR#570] [Problems]
I just got a new Macbook Pro M1 Pro with Monterey IOS 12.0.1 and Midi2lr isn't working properly. I use a Behringer BC/R2000 controller. I have midi2lr v5.2.0.0 installed.

Lightroom doesn't seem to be telling the BCR2000 what values are already set for each image. As I select images in LR, the levels of the knobs on the controller do not update. When I turn a knob, the image adjustment will suddenly jump to whatever value the knob was left at. This is annoying.

How do I force midi2lr to update the controller when I move between images in LR?

If I connect the controller to my old mac, it works fine. As soon as I connect it to my new mac, lightroom no longer updates the controller knob values when I switch between images.

This is a new MacBook and I installed midi2lr right after installing a fresh install of Lightroom.

Thank you!

Jim Vetter

[MIDI2LR.log](https://github.com/rsjaffe/MIDI2LR/files/7644488/MIDI2LR.log)
[MIDI2LRinfo.txt](https://github.com/rsjaffe/MIDI2LR/files/7644489/MIDI2LRinfo.txt)

**Comment by rsjaffe** [comment]
This is a problem with the Behringer driver, not with MIDI2LR. See https://github.com/rsjaffe/MIDI2LR/discussions/565#discussioncomment-1717967 For how another person solved it.
